interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/component/defaultconfig/DefaultComponentConfigDeterminer.scala

```diff
@@ -0,0 +1,62 @@
+package pl.touk.nussknacker.engine.definition.component.defaultconfig
+
+import pl.touk.nussknacker.engine.api.component.{ComponentGroupName, ComponentInfo, SingleComponentConfig}
+import pl.touk.nussknacker.engine.definition.component._
+
+object DefaultComponentConfigDeterminer {
+
+  def forNotBuiltInComponentType(
+      componentTypeSpecificData: ComponentTypeSpecificData,
+      hasReturn: Boolean
+  ): SingleComponentConfig = {
+    // TODO: use convention icon = componentGroup instead of code
+    val configData = componentTypeSpecificData match {
+      case SourceSpecificData =>
+        ComponentConfigData(DefaultsComponentGroupName.SourcesGroupName, DefaultsComponentIcon.SourceIcon)
+      case SinkSpecificData =>
+        ComponentConfigData(DefaultsComponentGroupName.SinksGroupName, DefaultsComponentIcon.SinkIcon)
+      case ServiceSpecificData if hasReturn =>
+        ComponentConfigData(DefaultsComponentGroupName.EnrichersGroupName, DefaultsComponentIcon.EnricherIcon)
+      case ServiceSpecificData =>
+        ComponentConfigData(DefaultsComponentGroupName.ServicesGroupName, DefaultsComponentIcon.ServiceIcon)
+      case CustomComponentSpecificData(_, true) =>
+        ComponentConfigData(
+          DefaultsComponentGroupName.OptionalEndingCustomGroupName,
+          DefaultsComponentIcon.CustomComponentIcon
+        )
+      case CustomComponentSpecificData(_, _) =>
+        ComponentConfigData(DefaultsComponentGroupName.CustomGroupName, DefaultsComponentIcon.CustomComponentIcon)
+      case _ =>
+        throw new IllegalStateException(
+          s"InitialComponentConfigDeterminer used with non model component: $componentTypeSpecificData"
+        )
+    }
+    SingleComponentConfig(
+      params = None,
+      icon = Some(configData.icon),
+      docsUrl = None,
+      componentGroup = Some(configData.componentGroup),
+      componentId = None
+    )
+  }
+
+  def forBuiltInComponent(info: ComponentInfo): SingleComponentConfig = SingleComponentConfig(
+    params = None,
+    Some(DefaultsComponentIcon.forBuiltInComponent(info)),
+    // TODO: move from defaultModelConfig.conf to here + convention instead of code
+    docsUrl = None,
```

Comment:
WDYT e.g. https://nussknacker.io/documentation/docs/next/scenarios_authoring/BasicNodes/#filter? We have some documentation for BultIn / Basic components.

Reply:
Take a look at TODO above, we already have these links in the `defaultModelConfig.conf`. I wanted to keep this change small. I'll fix it in the next PR